zebra/ioctl.c

```diff
@@ -460,10 +462,12 @@ struct in6_ifreq {
 	int ifr6_ifindex;
 };
 #endif /* _LINUX_IN6_H */
-
 /* Interface's address add/delete functions. */
 int if_prefix_add_ipv6(struct interface *ifp, struct connected *ifc)
 {
+#ifdef HAVE_NETLINK
+	return kernel_address_add_ipv6 (ifp, ifc);
+#else /* ! HAVE_NETLINK */
 	int ret;
```

Comment:
@pguibert6WIND I think this is ok, but I'm wondering if we need to support both Netlink and ioctl here. We are already under a `#ifdef LINUX_IPV6` block, so checking for `HAVE_NETLINK` is kind of redundant (unless someone tries to run FRR on Linux 2.0 from 1996, when Netlink didn't exist yet). My suggestion would be to completely remove the call to `if_ioctl_ipv6()` and associated code.